### PR TITLE
Pasteup2016: Copy guss-layout to Pasteup2016

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

removes guss-layout
## Request for comment

@sammorrisdesign 
## 

_Does this PR meet the [contributing guidelines](https://github.com/guardian/frontend/blob/issue_pr_templates/.github/CONTRIBUTING.md#submission)?_

### CODE REVIEW

File: static/src/stylesheets/_head.common.scss [hunk 1]
```diff
@@ -36,20 +36,7 @@
 @import 'layout/_ab-new-header-test-variant';
 @import 'layout/_helpers';
 @import 'layout/_side-margins';
-@include guss-row-utility($detect: true);
-@include guss-columns(
```
Comment (sndrs) [inline static/src/stylesheets/_head.common.scss]:
```
Searching 7292 files for "l-columns"

0 matches
```

File: static/src/stylesheets/pasteup/_src.scss [hunk 2]
```diff
@@ -1,2 +1,6 @@
+$browser-supports-columns: true !default;
+$browser-supports-flexbox: true !default;
```
Comment (sammorrisdesign) [inline static/src/stylesheets/pasteup/_src.scss]: Why add these vars here?

Comment (sndrs) [inline static/src/stylesheets/pasteup/_src.scss]: these are globals really, not unique to these modules. feel like they should be elevated out of individual components?

Comment (sammorrisdesign) [inline static/src/stylesheets/pasteup/_src.scss]: :+1: Although I've merged mine, so you got conflicts lol

**Pull request merged**